src/core/struct_tree.js

```diff
@@ -322,6 +355,11 @@ class StructTreePage {
             type: "object",
             id: kid.refObjId,
           });
+        } else if (kid.type === StructElementType.ANNOTATION) {
+          obj.children.push({
+            type: "object",
+            id: `pdfjs_internal_id_${kid.refObjId}`,
```

Comment:
This uses a quite different id-format compared to the ones above, is that actually intended?
*Ninja-edit:* I'm *guessing* that it might be related to the following constant, in which case we really ought to use that here (after moving the definition into the `src/shared/util.js` file):
https://github.com/mozilla/pdf.js/blob/92f7653cfbc2b8e824991f496564cf17fc3ee09b/src/display/display_utils.js#L34